content/docs/start/experiments.md

```diff
@@ -19,31 +19,82 @@ once they're no longer needed.
 ## Running experiments
 
 In the previous page, we learned how to tune
-[ML pipelines](/doc/start/data-pipelines) and compare the changes. Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+[ML pipelines](/doc/start/data-pipelines) and compare the changes. In this
+section we'll use a [new Get Started project][dvcgs] to illustrate
+experimentation features in DVC 2.0
 
 `dvc exp run` makes it easy to change <abbr>hyperparameters</abbr> and run a new
-experiment:
+experiment. We'll use it to find parameters that results in better
+classification performance for [Fashion-MNIST][fmnist] dataset.
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
+
+> You can run these commands in the container we built for this tutorial. It has
+> all the code and data required to run these examples.
+> `docker run -it dvcorg/doc-start:experiments`
+
+In order to run a baseline experiment with the default parameters defined in
+`params.yaml`:
+
+```dvc
+dvc exp run
+```
+
+This resembles `dvc repro` without any command-line arguments. However, when
+using `dvc repro` we need to update `params.yaml` manually, run the pipeline, if
+the results are worth it commit them to DVC and Git. `dvc exp` automates this
+process through its subcommands.
+
+Let's see some metrics produced by this baseline experiment:
+
+```dvc
+$ dvc exp show --include-metrics categorical_accuracy,precision,recall \
+               --include-params  model.name,model.cnn.conv_units
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┓
```

Comment:
should we use screenshots here? tables are not colored, the probably do not fit well?

Reply:
probably it's better to use the repo itself like we do with other images here

Reply:
There were some discussions around keeping PR size minimal. I prefer to put them into this repository as well.